### DISCUSSION TITLE
Changing DocumentRoot on Ubuntu22.04

### DISCUSSION THREAD

**Discussion by web1928** [Homebrew/discussions#5147] [Linux]
### Additional Info

- [ ] I am using Windows Subsystem for Linux

### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.9
ORIGIN: https://github.com/Homebrew/brew
HEAD: e5fefd73cd97cd36ae3af29551f529ae59b333d6
Last commit: 3 days ago
Core tap JSON: 21 Feb 21:24 UTC
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :0
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 3.1.4 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: quad-core 64-bit skylake
Clang: N/A
Git: 2.34.1 => /bin/git
Curl: 7.81.0 => /bin/curl
Kernel: Linux 6.5.0-18-generic x86_64 GNU/Linux
OS: Ubuntu 22.04.4 LTS (jammy)
Host glibc: 2.35
/usr/bin/gcc: 11.4.0
/usr/bin/ruby: N/A
glibc: N/A
gcc@11: N/A
gcc: 13.2.0
xorg: N/A
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Hey guys. I am fairly new to Brew and Linux and have recently installed on my Ubuntu22.04 desktop.

I removed the native apache2 installation and have installed httpd/php/mariadb, just as I have on my macbook. However, I don;t seem to be able to change the httpd DocumentRoot from `/home/linuxbrew/.linuxbrew/var/www` to `/home/myname/Sites`.

If I try to make the change in the `httpd.conf file`, httpd just doesn't start as a service. If I try and start with `brew services start httpd`, it says started, but when I check, it is not running.

I'm not sure if this is a permissions issue or there is an additional config change I need to make. I have tried searches in Google, but not really returning anything on brew>httpd>ubuntu.

Thanks in advance for any suggestions.

/D

**Comment by SMillerDev** [comment]
Did you try running the manual command in `brew info httpd` to see if it shows any errors?
